resources/splunk_install.rb

```diff
@@ -3,26 +3,22 @@
 #
 # Resource for managing the installation of Splunk
 class SplunkInstall < ChefCompat::Resource
-  include CernerSplunk::PlatformHelpers, CernerSplunk::PathHelpers
+  include CernerSplunk::PlatformHelpers, CernerSplunk::PathHelpers, CernerSplunk::ResourceHelpers
 
   resource_name :splunk_install
 
   property :name, String, name_property: true, identity: true
   property :package, [:splunk, :universal_forwarder], required: true
   property :version, String, required: true
   property :build, String, required: true
+  property :user, String, default: lazy { node['current_user'] || package == :splunk ? 'splunk' : 'splunkforwarder' }
```

Comment:
I thought it was possible to run chef-client as not root?
I suppose it wouldn't hurt anything to run forwarders as the splunk user, too.